examples/connext_dds/remote_procedure_call/py/inventory_service.py

```diff
@@ -0,0 +1,95 @@
+#
+# (c) 2023 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+#
+# RTI grants Licensee a license to use, modify, compile, and create derivative
+# works of the Software solely for use with RTI products.  The Software is
+# provided "as is", with no warranty of any type, including any warranty for
+# fitness for any purpose. RTI is under no obligation to maintain or support
+# the Software.  RTI shall not be liable for any incidental or consequential
+# damages arising out of the use or inability to use the software.
+#
+
+import argparse
+import rti.connextdds as dds
+import rti.rpc as rpc
+import rti.asyncio
+from asyncio import sleep
+
+from Inventory import (
+    Item,
+    InventoryContents,
+    InventoryService,
+    UnknownItemError,
+)
+
+
+class InventoryImpl(InventoryService):
+    """Implements a simple InventoryService"""
+
+    def __init__(self, delay: int = 0):
+        """Creates an InventoryImpl with an few initial items"""
+
+        self.delay = delay
```

Comment:
Not sure if this is intentional, but I noticed there isn't any delay option in the C++11 version

Reply:
Done